subprojects/gst-plugins-bad/sys/msdk/gstmsdkenc.c

```diff
@@ -82,15 +82,13 @@ GST_DEBUG_CATEGORY_EXTERN (gst_msdkenc_debug);
 static GstStaticPadTemplate sink_factory = GST_STATIC_PAD_TEMPLATE ("sink",
     GST_PAD_SINK,
     GST_PAD_ALWAYS,
-    GST_STATIC_CAPS (GST_MSDK_CAPS_STR
-        ("{ NV12, I420, YV12, YUY2, UYVY, BGRA }", "NV12"))
```

Comment:
According to media-driver https://github.com/intel/media-driver#decodingencoding-features, 8bit/422 is supported for hevc encoder from platforms > ICL, so we may include yuy2 in the caps and add a platform check somewhere else (like in gst_msdkenc_init_encoder ? ).